torchok/tasks/detection.py

```diff
@@ -48,13 +48,18 @@ def __init__(
         self.num_scales = num_scales or len(self.backbone.out_encoder_channels)
 
         # NECK
-        neck_params = neck_params or dict()
-        neck_in_channels = self.backbone.out_encoder_channels[-self.num_scales:][::-1]
-        self.neck = DETECTION_NECKS.get(neck_name)(in_channels=neck_in_channels, **neck_params)
+        if neck_name is not None:
+            neck_params = neck_params or dict()
+            neck_in_channels = self.backbone.out_encoder_channels[-self.num_scales:][::-1]
+            self.neck = DETECTION_NECKS.get(neck_name)(in_channels=neck_in_channels, **neck_params)
+            head_in_channels = self.neck.out_channels
+        else:
+            self.neck = nn.Identity()
+            head_in_channels = self.backbone.out_encoder_channels[-self.num_scales:][::-1]
 
         # HEAD
         head_params = head_params or dict()
-        self.bbox_head = HEADS.get(head_name)(in_channels=self.neck.out_channels, **head_params)
+        self.bbox_head = HEADS.get(head_name)(joint_loss=self.losses, in_channels=head_in_channels, **head_params)
```

Comment:
So we send joint loss to bbox_head, ok

Reply:
Add comment please that some detection head can use joint loss parameters and change it